pkg/kubelet/container/runtime_cache_fake.go

```diff
@@ -30,7 +30,7 @@ type TestRuntimeCache struct {
 func (r *TestRuntimeCache) UpdateCacheWithLock() error {
 	r.Lock()
 	defer r.Unlock()
-	return r.updateCache(context.Background())
+	return r.updateCache(context.TODO())
```

Comment:
Please, pass `ctx` into `UpdateCacheWithLock` and use it here.

Reply:
done, thx~